yang/frr-filter.yang

```diff
@@ -0,0 +1,365 @@
+module frr-filter {
+  yang-version 1.1;
+  namespace "http://frrouting.org/yang/filter";
+  prefix frr-filter;
+
+  import ietf-inet-types {
+    prefix inet;
+  }
+  import ietf-yang-types {
+    prefix yang;
+  }
+
+  organization "Free Range Routing";
+  contact
+    "FRR Users List:       <mailto:frog@lists.frrouting.org>
+     FRR Development List: <mailto:dev@lists.frrouting.org>";
+  description "This module defines route map settings";
+
+  revision 2019-07-04 {
+    description "Initial revision";
+  }
+
+  /*
+   * Types.
+   */
+  typedef access-list-standard {
+    description "Standard IPv4 access list (any, host or a prefix)";
+    type uint16 {
+      range "1..99 | 1300..1999";
+    }
+  }
+
+  typedef access-list-extended {
+    description
+      "Extended IPv4 access list (source / destination any, hosts or prefixes)";
+    type uint16 {
+      range "100..199 | 2000..2699";
+    }
+  }
+
+  typedef access-list-legacy {
+    description "Standard/Extended IPv4 access list";
+    type uint16 {
+      range "1..199 | 1300..2699";
```

Comment:
This range overlaps with "100..199 | 2000..2699" from `access-list-extended`. Is this correct?

Reply:
Yes. They overlap because it is the union of standard and extended legacy access lists (using numbers instead of name).